app/models/profile.rb

```diff
@@ -50,6 +50,10 @@ def from_openscap_parser(op_profile, benchmark_id: nil, account_id: nil)
     end
   end
 
+  def ssg_version
+    benchmark.version
```

Comment:
Observation: We use similar attribute in API(s). Mind to refactor?

Reply:
This was as simple as removing the ssg_version in the profile graphql type :)